jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractListPresenter.java

```diff
@@ -133,6 +133,7 @@ public void updateDataOnCallback(List<T> instanceSummaries,
                                    instanceSummaries);
 
         updateRefreshTimer();
+        getListView().getListGrid().setVisibleSelectedItems();
```

Comment:
the same should be applied in the _updateDataOnCallback_ method above. My suggestion would be to consolidate both methods, the one receiving a _PageResponse_ should call this one passing the related parameters.

Reply:
done!